packages/@aws-cdk/aws-cloudwatch/lib/graph.ts

```diff
@@ -223,6 +222,23 @@ export interface GraphWidgetProps extends MetricWidgetProps {
    * @default false
    */
   readonly setPeriodToTimeRange?: boolean;
+
+  /**
+   * The default period, in seconds, for all metrics in this widget.
+   * The period is the length of time represented by one data point on the graph.
+   * This default can be overridden within each metric definition.
+   *
+   * @default cdk.Duration.seconds(300)
+   */
+  readonly period?: cdk.Duration;
+
+  /**
+   * The default statistic to be displayed for each metric.
+   * This default can be overridden within the definition of each individual metric
+   *
+   * @default - The statistic for each metric is used
```

Comment:
confusing statements?!

I'm guessing this is the default stat if no stat is specified on the individual metrics. What's the default if no stat is specified on the metric and here?

Reply:
Honestly, not sure. It doesn't appear to be documented. In the console, if you remove all stats from the source, it _appears_ to default to Average. However, that could be console-specific behavior.